src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -23,42 +22,38 @@ interface Props {
 }
 
 function ProductGallery({ title, slug }: Props) {
+  const [isFilterOpen, setIsFilterOpen] = useState<boolean>(false)
   const { pages, state: searchState, addNextPage, addPrevPage } = useSearch()
   const { data } = useGalleryQuery()
-
   const totalCount = useTotalCount(data)
-
-  const { next, prev } = usePagination(totalCount)
-  const { isMobile } = useWindowDimensions()
-
-  const [isFilterOpen, setIsFilterOpen] = useState<boolean>(false)
-
   const orderedFacets = useOrderedFacets(data)
+  const { next, prev } = usePagination(totalCount)
 
   return (
-    <>
-      <div className="product-listing__filters">
-        <Filter
-          slug={slug}
-          isOpen={isFilterOpen}
-          facets={orderedFacets}
-          onDismiss={() => setIsFilterOpen(false)}
-        />
-      </div>
+    <div className="product-listing / grid-content-full">
+      <div className="product-listing__content-grid / grid-content">
+        <div className="product-listing__filters">
+          <Filter
+            slug={slug}
+            isOpen={isFilterOpen}
+            facets={orderedFacets}
+            onDismiss={() => setIsFilterOpen(false)}
+          />
+        </div>
 
-      <div
-        className="product-listing__results-count"
-        data-testid="total-product-count"
-        data-count={totalCount}
-      >
-        <h2>{totalCount} Results</h2>
-      </div>
+        <div
+          className="product-listing__results-count"
+          data-testid="total-product-count"
+          data-count={totalCount}
+        >
+          <h2>{totalCount} Results</h2>
+        </div>
 
-      <div className="product-listing__sort">
-        <Sort />
+        <div className="product-listing__sort">
+          <Sort />
 
-        {isMobile && (
           <Button
+            className="button display-mobile"
```

Comment:
```suggestion
         
```

Reply:
I think we can remove this line and add the code below on [product-gallery.scss](https://github.com/vtex-sites/base.store/pull/328/files#diff-70ef4213089f31f7dd2bca7c7128d8b279d93fee490fd5ee08fa460a419f0354) after line 70:
 
```
.button {
  display: none;
}
```